kamailio/default.cfg

```diff
@@ -618,6 +621,8 @@ route[EXTERNAL_TO_INTERNAL_RELAY]
 
 onreply_route[EXTERNAL_REPLY]
 {
+    # this route handles replies that are coming from a carrier or customer device
+    xlog("L_INFO", "$ci|start|received external $rm reply $T_reply_code $rr source $si:$sp");
     xlog("L_INFO", "$ci|log|external reply $T_reply_code");
```

Comment:
Karl's comments I believe on an earlier stand up indicated the desire to not modify current log lines to not take a chance at breaking current apps like fail2ban. My first proposal that was in a gist (now deleted) did replace instead of add.